docs/super-commands.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Using Super Commands with the Flow CLI
+sidebar_title: Super Commands
+description: How Flow super commands work
+---
+
+Flow CLI Super commands are set of commands that can be used during development 
+of your dApp to greatly simplify the workflow. The result is you can focus on writing the contracts 
+and the commands will take care of the rest. 
+
+
+## Setup
+Flow setup is the initial command you run to start your new Flow project. By default, it will create 
+basic folder structure and a flow.json configuration. 
+
+Running the command:
+```
+> flow setup
+```
+
+It will contain the following resources:
+- `/contracts` folder should contain all your Cadence contracts,
+- `/scripts` folder should contain all your Cadence scripts,
+- `/transactions` folder should contain all your Cadence transactions,
+- `/tests` folder should contain all your Cadence tests,
+- `flow.json` is a configuration file for your project, which will be automatically maintained.
+
+### Using Scaffolds
+Based on the purpose of your project you can select from a list of available scaffolds. 
+You can access the scaffolds by simply using the `--scaffold` flag like so:
+```
+> flow setup --scaffold
+```
+
+The list of scaffolds will continuously grow, and you are welcome to contribute to that. 
+You can contribute by creating your own scaffold repository which can then be added to the scaffold 
+list by opening a PR here. [TODO]
+
+
+## Develop
+After creating the project using the flow setup command you should start the emulator in 
+the project directory by running `flow emulator`. After emulator is started up you can continue by 
+running the flow develop command like so:
+```
+> flow dev
+```
+This will continuously watch for your project Cadence files and keep the deployed on emulator. 
```

Comment:
```suggestion
This will continuously watch for your projects Cadence files for changes and keep them in sync with the deployed contracts on the emulator. 
```

This is also repeated below.